roles/zabbix_agent/defaults/main.yml

```
@@ -24,6 +24,7 @@ zabbix_agent_forceactivechecksonstart: false
 zabbix_agent_heartbeatfrequency: 60
 zabbix_agent_hostname: "{{ inventory_hostname }}"
 zabbix_agent_include_mode: "0750"
+zabbix_agent_additional_includes: []
```

```suggestion
zabbix_agent_additional_include: []
```
Please decide whether it is `zabbix_agent_additional_includes` or `zabbix_agent_additional_include`. I would go with `zabbix_agent_additional_includes`, i.e. change this variable in other files but up to you.